mews-operations/reservations.md

```diff
@@ -288,12 +322,29 @@ The third `reservation` definition shows the partial cancellation - cancelling t
 | `12` | China Unionpay |
 | `13` | Maestro |
 
-#### Channel
+#### Source
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `code` | `int` | required | [Channel](#channels) code. |
-| `name` | `string` | required | Name of the Channel. |
+| `code` | `int` | required \(exc. new unknown sources\) | Source code. |
```

Comment:
| `code` | `int` | required \(exc. new unknown sources\) | Code number of the source or channel. |